tools/PingEnumerator.py

```diff
@@ -1,57 +1,76 @@
-#!/usr/bin/python -u
```

Comment:
Why did you change this?
It seems the opposite of your suggestion here:
https://github.com/bluerobotics/companion/pull/139#discussion_r229822133

Reply:
Why is it the opposite ?
> #!/usr/bin/env bash
should be default

I moved from `/usr/bin/python` to `/usr/bin/env python`.

Reply:
am I going crazy?